src/test/java/hudson/plugins/logparser/LogParserPublisherTest.java

```diff
@@ -0,0 +1,49 @@
+package hudson.plugins.logparser;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.jenkinsci.plugins.workflow.structs.DescribableHelper;
+import org.junit.Test;
+
+import jenkins.tasks.SimpleBuildStep;
+
+public class LogParserPublisherTest {
+    
+    @Test
+    public void testWorkflowInstantiation() throws Exception {
+        Map<String, Object> m = getInstanceParameters();
+        
+        Object inst = (LogParserPublisher) DescribableHelper.instantiate(LogParserPublisher.class.asSubclass(SimpleBuildStep.class), m);
```

Comment:
This is to test the creation from workflow without running Jenkins. If there is a better way to do it (like using Groovy exactly like in workflow) without invoking the LogParserPublisher itself (could not find a way in the code) please tell me how. Or you could ask your colleagues from cloudbees who are implementing it.